### PR TITLE
frontend: bump flag-icons dep. to latest 7.2.3

### PR DESCRIPTION
**Pull request by NicolaLS**
no breaking changes, simple update.

flag-icons [CHANGELOG.md](https://github.com/lipis/flag-icons/blob/main/CHANGELOG.md)

### CODE REVIEW

**Comment by NicolaLS** [comment]
(tested on `webdev` chrome)
flags-screenshot

File: frontends/web/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@walletconnect/jsonrpc-types": "^1.0.3",
     "@walletconnect/types": "^2.10.4",
     "@walletconnect/web3wallet": "^1.9.3",
-    "flag-icons": "^6.6.6",
+    "flag-icons": "^7.2.3",
```
Comment (thisconnect) [inline frontends/web/package.json:27]: please use `--save-exact` see https://github.com/BitBoxSwiss/bitbox-wallet-app?tab=readme-ov-file#npm-dependencies

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/package.json (hunk 1 above)
Comment (NicolaLS) [reply frontends/web/package.json]: done/rebased @thisconnect

**Review by thisconnect — APPROVED** [review]
thanks tested LGTM

(FYI: I usually check Nepal flag as it has a special none-rectangle form)

**Pull request merged**